src/sentry/search/events/fields.py

```diff
@@ -112,6 +113,45 @@ def key_transaction_expression(user_id, organization_id, project_ids):
     ]
 
 
+def team_key_transaction_expression(organization_id, team_ids, project_ids):
+    if organization_id is None or team_ids is None or project_ids is None:
+        raise InvalidSearchQuery("Missing necessary meta for key transaction field.")
+
+    team_key_transactions = (
+        TeamKeyTransaction.objects.filter(
+            organization_id=organization_id,
+            project_id__in=project_ids,
+            team_id__in=team_ids,
+        )
+        .order_by("transaction", "project_id")
+        .values("project_id", "transaction")
+        .distinct("transaction", "project_id")
+    )
+
+    if not len(team_key_transactions):
```

Comment:
nit: but why not just check its 0 instead of going with a falsey check?
```suggestion
    if len(team_key_transactions) == 0:
```